atom/lib/import-js.js

```diff
@@ -0,0 +1,302 @@
+'use strict';
+
+const AskForSelectionView = require('./AskForSelectionView');
+const getEnv = require('./getEnv');
+
+const BufferedProcess = require('atom').BufferedProcess;
+const CompositeDisposable = require('atom').CompositeDisposable;
```

Comment:
They could if Node had destructuring or if we used Babel or something to transpile. Node 6 is supposed to include destructuring.